commands/playground/youtubeQueue.js

```diff
@@ -0,0 +1,30 @@
+const ytdl = require("ytdl-core");
+const {youtubeSearch}  = require("./../music/youtubeSearch");
+
+const youtubeQueue = (queue, connection, voiceChannel) => {
+    
+    youtubeSearch(queue[0]).then((ytvid) => {
```

Comment:
good use of promises

ytvid -> ytVid

if queue[0] is used up here, maybe you should remove it here and not later -- maybe shift here.
maybe should check length > 0

Reply:
Fixed